integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/timeseries/TimeSeriesUtil.scala

```diff
@@ -54,6 +56,107 @@ object TimeSeriesUtil {
     }
   }
 
+  def getGranularityKey(dmProperties: Map[String, String]): String = {
```

Comment:
why to return the same value? if required only for validation, can change method name and return true/false

Reply:
I need know what is the key, and then remove it for updatedDmProperties 

 val updatedDmProperties = dmproperties - TimeSeriesUtil.getGranularityKey(dmproperties)